src/Sdk/Client.php

```diff
@@ -0,0 +1,74 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\SdkBlueprint\Sdk;
+
+use GuzzleHttp\Client as GuzzleClient;
+use GuzzleHttp\Exception\RequestException;
+use LoyaltyCorp\SdkBlueprint\Sdk\Exceptions\InvalidRequestDataException;
+use LoyaltyCorp\SdkBlueprint\Sdk\Interfaces\ClientInterface;
+use LoyaltyCorp\SdkBlueprint\Sdk\Interfaces\RequestInterface;
```

Comment:
|`line 36`|
|---|
Please make sure that all your optional parameters have `= null`.